.git-blame-ignore-revs

```diff
@@ -1,3 +1,2 @@
 # Migrate code style to Black
 de6a33a03040e370fc5701622cb2496c7ece4cff
```

Comment:
Why remove this?

Reply:
It's there it's just fixing up the end of file. Weird that it's formatted that way though.

Reply:
Huh. Weird. Github made it look like the hash was removed.